components/kyma-environment-broker/internal/broker/instance_create.go

```diff
@@ -76,20 +83,22 @@ func NewProvision(cfg Config,
 	}
 
 	return &ProvisionEndpoint{
```

Comment:
Does it make sensible to apply new validation to other operations as well?

Reply:
Don't get it. We need to talk.

Reply:
Discussed in private communication.